AnnService/inc/Core/Common/cuda/KNN.hxx

```diff
@@ -966,19 +480,18 @@ auto after_copy = std::chrono::high_resolution_clock::now();
 
 auto end_t = std::chrono::high_resolution_clock::now();
 
-  LOG(SPTAG::Helper::LogLevel::LL_Info, "Total times - prep time:%0.3lf, tree build:%0.3lf, neighbor compute:%0.3lf, Copy results:%0.3lf, Total runtime:%0.3lf\n", prep_time, tree_time, KNN_time, D2H_time, ((double)std::chrono::duration_cast<std::chrono::seconds>(end_t - start_t).count()) + (((double)std::chrono::duration_cast<std::chrono::milliseconds>(end_t - start_t).count())/1000));
+  LOG(SPTAG::Helper::LogLevel::LL_Info, "Total times - prep time:%0.3lf, tree build:%0.3lf, neighbor compute:%0.3lf, Copy results:%0.3lf, Total runtime:%0.3lf\n", prep_time, tree_time, KNN_time, D2H_time, GET_CHRONO_TIME(start_t, end_t));
 
   for(int gpuNum=0; gpuNum < NUM_GPUS; ++gpuNum) {
     tptrees[gpuNum]->destroy();
     delete tptrees[gpuNum];
     cudaFree(d_tptrees[gpuNum]);
-    cudaFree(d_points[gpuNum]);
```

Comment:
Do we need to free/delete d_pointset?

Reply:
Good catch, added missing free statements